lib/ansible/config/base.yml

```diff
@@ -569,17 +604,31 @@ DEFAULT_CALLBACK_PLUGIN_PATH:
   - {key: callback_plugins, section: defaults}
   type: pathspec
   yaml: {key: plugins.callback.path}
-DEFAULT_CALLBACK_WHITELIST:
-  name: Callback Whitelist
+DEFAULT_CALLBACK_ENABLED:
```

Comment:
For whatever reason, the `ENABLED` terminology makes more sense to me here than above, though it's nice to have the `LIST` in the key. Same thing with ditching the `DEFAULT` prefix, since we have to have a new key anyway. Seems like `CALLBACK_ENABLED_LIST` or `CALLBACK_ALLOWLIST`- the former feels better, but the latter is more consistent. (shrug)

Reply:
callbacks_enabled?

Reply:
Still sounds awfully "boolean" to me- it's not clear that it should be a list of callback names